Formula/spdx-sbom-generator.rb

```diff
@@ -0,0 +1,36 @@
+class SpdxSbomGenerator < Formula
+  desc "Support CI generation of SBOMs via golang tooling"
+  homepage "https://github.com/spdx/spdx-sbom-generator"
+  url "https://github.com/spdx/spdx-sbom-generator/archive/refs/tags/v0.0.13.tar.gz"
+  sha256 "7d088f136a53d1f608b1941362c568d78cc6279df9c1bdb3516de075cb7f10c3"
+  license any_of: ["Apache-2.0", "CC-BY-4.0"]
+
+  depends_on "go" => [:build, :test]
+
+  resource "self" do
+    url "https://github.com/spdx/spdx-sbom-generator/archive/refs/tags/v0.0.13.tar.gz"
+    sha256 "7d088f136a53d1f608b1941362c568d78cc6279df9c1bdb3516de075cb7f10c3"
+  end
```

Comment:
Not really a fan of this. Can we try to mock a simple project in the `test` block and then use `spdx-sbom-generator` on that?

Reply:
How do you mean? Should we pull something else down?

Reply:
I'm sorry I am not sure how a mock would work.

Reply:
make an empty requirements.txt file and then run the tool to see if it works.

Reply:
Ahh ok I will try that.

Reply:
So that did not work as the spdx-sbom-generator needs pyenv if there is a requirements.txt file and the requirements.txt file has to explicitly tell it what version of python is required. :/

Reply:
Pick another language with less requirements.

Reply:
Ok, so from my testing it will never create an output without listing dependencies. I tested nodejs, golang, python, and rust. Empty projects in all these languages throw errors with the tool.

Reply:
That's really weird, I would expect "no materials" to be a supported "bill of materials". Maybe just add a single dependency then? It doesn't have to be an empty project to test, we just want to avoid pulling in a whole new repo.

Reply:
We can also create an empty project and then check that we get the error we expect.